docker/docker-compose-base.yml

```diff
@@ -46,7 +46,7 @@ services:
     ports:
       - ${MYSQL_PORT}:3306
     volumes:
-      - mysql_data:/var/lib/mysql
+      - ./mysql_data:/var/lib/mysql
```

Comment:
`mysql_data` is a volume defined later in this yaml. It's mounted from a docker managed directory.
In rare case if you want to mount from some specific host, change the volume definion there for yourself.